src/test/scala-2/chiselTests/BoringUtilsSpec.scala

```diff
@@ -34,12 +35,10 @@ class BoringUtilsSpec extends ChiselFlatSpec with ChiselRunners with Utils with
     BoringUtils.addSink(b, "x")
   }
 
-  behavior.of("BoringUtils.{addSink, addSource}")
+  behavior.of("BoringUtils.addSink and BoringUtils.addSource")
 
   it should "connect two wires within a module" in {
-    runTester(
-      new ShouldntAssertTester { val dut = Module(new BoringInverter) }
-    ) should be(true)
+    simulate(new ShouldntAssertTester { val dut = Module(new BoringInverter) })(Stimulus.runUntilFinished(3))
```

Comment:
This is neat.

Reply:
This is what I was primarily concerned about. Two alternatives to this API are a dedicated `simulateWithTimeout` or a default argument `timeout: BigInt` to this method. I liked the factoring in this patch better as it seemed like keeping all stimulus separate was cleaner.